EIPS/eip-2633.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2633
+title: Formal Proposal for Upgradable Governance
+author: Edson Ayllon (@edsonayllon)
+discussions-to: https://ethereum-magicians.org/t/eip-2633-formal-proposal-for-upgradable-governance/4264
+status: Draft
+type: Meta
+created: 2020-05-06
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+A formal proposal to set a precedent that Ethereum's governance structure can be changed and improved over time. 
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+
+In a survey done collecting feedback from EIP Editors, EIP Authors, and Core Developers, a need was signaled for a formal decision-making process for network upgrade EIP inclusion. While a formal decision-making process can be created, a proper governance solution may take years of research and funding for that research. This proposal is to allow to begin to create a formal decision-making process, while not committing to that formal decision-making process indefinitely, and encourage more improvements to decision making to be pursued and possibly funded.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+This proposal is to set a precedent for an upgradable governance system for Ethereum. The concept is that Ethereum's governance is not fixed, and can be improved on either regularly or as the need arises. Here, governance is defined as decision making. Currently, this would apply to network upgrade coordination (hard-forks), and not the EIP drafting process itself. 
+
+The initiating of an upgrade process can be done through motioning or by a schedule for example. The process of initiating a governance upgrade is unspecified in this EIP.
+
+The process of how improvements are made is also unspecified.
```

Comment:
EIPs are explicitly specifications, and this EIP doesn't specify anything so it isn't a valid EIP.

Reply:
Greg said the same thing a while back. I included specification with c48749c